CaloCluster/fcl/prolog.fcl

```diff
@@ -14,7 +14,17 @@ END_PROLOG
 BEGIN_PROLOG
 
 
-
+CaloClusterFast:
```

Comment:
can't this just live in the prolog_trigger file? currently, you have the same blocks defined on both files.

Reply:
Ok, will see what I can do

Reply:
Done